### PR TITLE
silder component

### PR DESCRIPTION
**Pull request by modrayyes**
image

### CODE REVIEW

File: client/src/components/slider/slider.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+function Slider() {
+  const [cities, setCities] = useState([]);
+
+  const getMainCities = () => {
+    const url = '/api/v1/';
+    return http
+      .get(url)
+      .then((response) => response)
+      .catch((err) => {
+        throw new Error(`Error in fetching products ${err}`);
```
Comment (LinaYahya) [inline client/src/components/slider/slider.js:21]: turn to async await and message.error to clear text

File: client/src/App.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import 'antd/dist/antd.css';
+import Slider from './components/slider/slider';
+
+function App() {
+  return (
+    <div className="App">
+      <Slider />
```
Comment (LinaYahya) [inline client/src/App.js:8]: pull the last changes and import Slider in the home page

File: client/src/components/slider/slider.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
```
Comment (LinaYahya) [inline client/src/components/slider/slider.js:1]: rename file to index.js

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by Nizar7zak — APPROVED** [review]
Great Job

**Review by reemsh2000 — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**